app/models/answer.rb

```diff
@@ -1,3 +1,6 @@
 class Answer < ApplicationRecord
   belongs_to :question
+
+  scope :find_true_answer,
+        ->(id){where("is_answer is true and question_id = ?", id)}
```

Comment:
```suggestion
  scope :find_true_answer,
        ->{where "is_answer is true"}
```

chỗ gọi scope em tận dụng association đi em
`question.answers.find_true_answer`